cmd/root.go

```diff
@@ -50,96 +52,180 @@ const (
 	waitRemoteLoggerTimeout = time.Second * 5
 )
 
-// TODO better name - there are other command flags these are just ... non lib.Options ones :shrug:
-type commandFlags struct {
-	defaultConfigFilePath string
-	configFilePath        string
-	exitOnRunning         bool
-	showCloudLogs         bool
-	runType               string
-	archiveOut            string
-	quiet                 bool
-	noColor               bool
-	address               string
-	outMutex              *sync.Mutex
-	stdoutTTY, stderrTTY  bool
-	stdout, stderr        *consoleWriter
+type globalFlags struct {
+	configFilePath string
+	runType        string
+	quiet          bool
+	noColor        bool
+	address        string
+	logOutput      string
+	logFormat      string
+	verbose        bool
 }
 
-func newCommandFlags() *commandFlags {
-	confDir, err := os.UserConfigDir()
-	if err != nil {
-		logrus.WithError(err).Warn("could not get config directory")
-		confDir = ".config"
-	}
-	defaultConfigFilePath := filepath.Join(
-		confDir,
-		"loadimpact",
-		"k6",
-		defaultConfigFileName,
-	)
+type globalState struct {
```

Comment:
Although private, could benefit from a documentation comment detailing its reason to be, in which context its meant to be used etc...

Reply:
Duplicate commit?

Reply:
Nope 😸 just copy pasted it as it applies too :)

Reply:
Resolved by https://github.com/grafana/k6/pull/2410/commits/55cd43ddcd30401a5f656604561a2204761d659d